velox/vector/arrow/Bridge.h

```diff
@@ -148,4 +148,14 @@ VectorPtr importFromArrowAsOwner(
     memory::MemoryPool* pool =
         &velox::memory::getProcessDefaultMemoryManager().getRoot());
 
+/// Release function for ArrowArray. Arrow standard requires it to recurse down
+/// to children and dictionary arrays, and set release and private_data to null
+/// to signal it has been released.
+void bridgeRelease(ArrowArray* arrowArray);
```

Comment:
The name of this function is a bit too cryptic: facebook::velox::bridgeRelease. Would it be possible to rename to make it clear that it releases an Arrow array? Maybe, releaseArrowArray (and releaseArrowSchema for the next one)?

@Yuhta @pedroerp Jimmy, Pedro, what do you think?

Reply:
`releaseArrowArray` and `releaseArrowSchema` would be more proper I think

Reply:
Curious though why do we need this, because you can call `arrowArray->release(arrowArray)` to achieve it

Reply:
Renamed them to `releaseArrowArray` and `releaseArrowSchema`.
@Yuhta You're right. Velox already sets the release callback in `exportBase`, so I do not need to call them explicitly.